server/controllers/metricsController.js

```diff
@@ -0,0 +1,42 @@
+const { spawn } = require('child_process');
+const fetch = require('node-fetch');
+
+const metricsController = {};
+
+const PROM_URL = 'http://127.0.0.1:9090/api/v1/';
+
+metricsController.getMemory = async (req, res, next) => {
+  try {
+    // create query at current time
+    const currentDate = new Date().toISOString();
+    let query = `/query_range?query=sum(rate(node_memory_MemFree_bytes[2m]))`;
+    query += `&start=${currentDate}&end=${currentDate}&step=1m`;
+    let memArr;
+    // send query to prometheus for node memory usage
+    const data = await fetch(PROM_URL + query)
```

Comment:
If we're using `await` here, do we need the `.then`s? I would stick with one or the other rather than mixing and matching.